### PR TITLE
Add a ScheduledDowntime object defined type, refs#7218

### PR DESCRIPTION
**Pull request by cropalato**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -581,6 +582,26 @@ icinga2::object::notificationcommand { 'mail-service-notification':
 
 This oobject use the same parameter defined to `checkcommand`.
 
+####[`icinga2::object::sicheduleddownload`](id:object_scheduleddownload)
```
Comment (nickchappell) [inline README.md]: Typos

**Comment by nickchappell** [comment]
Hi @cropalato,

Sorry for letting this sit for so long. I made an inline comment about some typos, but the code itself looks good and works, so this PR should be good to merge after the little typos are fixed.

**Comment by nickchappell** [comment]
Just for reference, the dev.icinga.org issue is: [https://dev.icinga.org/issues/7218](https://dev.icinga.org/issues/7218)

**Comment by nickchappell** [comment]
Hi @cropalato,

I merged in some other PRs and this one won't apply cleanly. Would you be able to rebase your feature branch off of the current master branch and re-push?

**Comment by cropalato** [comment]
Done.

**Comment by nickchappell** [comment]
Hi @cropalato,

The patch still won't apply correctly. Did you rebase your feature branch off of the current master branch of this repo and push it?

I merged in your other PR, #42, so if you rebase the current master onto your feature branch, your changes in #42 shouldn't cause any problems (both this one and #42 have changes to the `README`).

**Comment by cropalato** [comment]
ok. I'll create a new pr.

**Comment by cropalato** [comment]
should be ok now.

**Comment by nickchappell** [comment]
Patch still didn't apply. I pulled the PR's branch down locally and fixed the merge conflicts in the README before merging back into master.

This is now merged. Thanks @cropalato!

**Pull request closed without merging**